child_compassion/wizard/child_description_de.py

```diff
@@ -128,184 +123,375 @@ def _get_school_info_de(cls, cr, uid, child, case_study, context=None):
+
+            props_en_m = [emp.value_en for emp in case_study.male_guardian_ids]
+            props_en_f = [
+                emp.value_en for emp in case_study.female_guardian_ids]
+            props_en_mf = list(set(props_en_m) & set(props_en_f))
+            props_en = [props_en_m, props_en_f, props_en_mf]
+
+            props_de_m = [emp.value_de for emp in case_study.male_guardian_ids]
+            props_de_f = [
+                emp.value_de for emp in case_study.female_guardian_ids]
+            props_de_mf = list(set(props_de_m) & set(props_de_f))
+            props_de_mf = props_de_mf + \
+                [False] * (len(props_en_mf) - len(props_de_mf))
+            props_de = [props_de_m, props_de_f, props_de_mf]
+
+            # Male job
+            string += cls._get_guardian_job_string(
+                cr, uid, child, props_en, props_de, m_g, f_g, 0, context)
+
+            # Female job
+            string += cls._get_guardian_job_string(
+                cr, uid, child, props_en, props_de, m_g, f_g, 1, context)
+
+            # Same job
+            string = cls._get_guardian_job_string(
+                cr, uid, child, props_en, props_de,
+                m_g, f_g, 2, context) or string
+        return string
+
+    @classmethod
+    def _get_mf_g(cls, cr, uid, child, m_g, f_g, context=None):
+        # Comments for this function in child_description_fr
+        mf_g = u''
+
+        if child.gender == 'M':
+            prefix_m = u'Sein'
+            prefix_f = u'Seine'
+        else:
+            prefix_m = u'Ihr'
+            prefix_f = u'Ihre'
+
+        if (f_g[0] == u'grandmother' and m_g[0] == u'grandfather'):
+            mf_g = u'{} Großeltern'.format(prefix_f)
+        elif (f_g[0] == u'mother' and m_g[0] == u'father'):
+            mf_g = u'{} Eltern'.format(prefix_f)
+        else:
+            mf_g = u'{} {} et {} {}'.format(prefix_m, m_g[1], prefix_f, f_g[1])
+
+        return mf_g
+
+    @classmethod
+    def _get_guardian_job_string(
+            cls, cr, uid, child, props_en, props_de,
+            m_g, f_g, type, context=None):
+        # Comments for this function in child_description_fr
+        string = u''
+
+        if(child.gender == 'M'):
+            prefix_f = u'Seine {}'.format(f_g[1] if f_g else u'Mutter')
+            prefix_m = u'Sein {}'.format(m_g[1]if m_g else u'Vater')
+        else:
+            prefix_f = u'Ihre {}'.format(f_g[1] if f_g else u'Mutter')
+            prefix_m = u'Ihr {}'.format(m_g[1]if m_g else u'Vater')
+
+        prefix_mf = cls._get_mf_g(
+            cr, uid, child, m_g, f_g, context) if f_g and m_g else None
+
+        prefix = [prefix_m, prefix_f, prefix_mf]
+
+        work_as = [
+            u'arbeitet als', u'arbeitet als', u'arbeiten als']
+        is_employed = [u'arbeitet', u'arbeitet', u'arbeiten']
+        is_unemployed = [
+            u"ist arbeitslos", u"ist arbeitslos", u"sind arbeitslos"]
+
+        job_tags_work_as = {

```

Comment:
Aren't these words translated? (I don't remember the object name, but I'm sure you do :) )